<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>M3U8Downloader</class>
 <widget class="QWidget" name="M3U8Downloader">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1000</width>
    <height>618</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>M3U8下载器</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="2" column="0">
    <widget class="QLabel" name="label_2">
     <property name="text">
      <string>名称</string>
     </property>
    </widget>
   </item>
   <item row="3" column="0">
    <widget class="QLabel" name="label_3">
     <property name="text">
      <string>保存路径</string>
     </property>
    </widget>
   </item>
   <item row="0" column="0">
    <widget class="QLabel" name="label">
     <property name="text">
      <string>网址</string>
     </property>
    </widget>
   </item>
   <item row="3" column="1">
    <widget class="LineEdit" name="le_savepath"/>
   </item>
   <item row="3" column="2">
    <widget class="PushButton" name="pb_choosepath">
     <property name="text">
      <string>选择</string>
     </property>
    </widget>
   </item>
   <item row="2" column="1" colspan="2">
    <widget class="LineEdit" name="le_name"/>
   </item>
   <item row="0" column="1" colspan="2">
    <widget class="LineEdit" name="le_url"/>
   </item>
   <item row="4" column="0" colspan="3">
    <widget class="PushButton" name="pb_download">
     <property name="text">
      <string>下载</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>PushButton</class>
   <extends>QPushButton</extends>
   <header>qfluentwidgets</header>
  </customwidget>
  <customwidget>
   <class>LineEdit</class>
   <extends>QLineEdit</extends>
   <header>qfluentwidgets</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
